package.json

```diff
@@ -50,7 +50,7 @@
   "dependencies": {
     "fs-extra": "^7.0.1",
     "nodeify": "^1.0.1",
-    "webfont": "^9.0.0"
+    "webfont": "file:./../webfont"
```

Comment:
Don't do this, i will take care about this after release

Reply:
I've noticed right after sending PR...